modules/ivc_champva/app/services/ivc_champva/metadata_validator.rb

```diff
@@ -28,6 +28,8 @@ def self.validate_last_name(metadata)
     end
 
     def self.validate_file_number(metadata)
+      return metadata if metadata['fileNumber'].blank?
+
       validate_presence_and_stringiness(metadata['fileNumber'], 'file number')
       unless metadata['fileNumber'].match?(/^\d{8,9}$/)
         raise ArgumentError, 'file number is invalid. It must be 8 or 9 digits'
```

Comment:
We have a few forms that don't have fileNumber but we don't want to fail the entire thing. Just added a guard statement and left the rest of he code as is.